mainhandler/imageregistryhandler.go

```diff
@@ -606,8 +606,10 @@ func (registryScan *registryScan) setRegistryKind() {
 }
 
 func (registryScan *registryScan) getRegistryProvider() string {
+	if registryScan.registryInfo.RegistryProvider != "" {
+		return registryScan.registryInfo.RegistryProvider
```

Comment:
the only thing that I'm concerned about when this is the first `if` statement is whether it will impact the other conditionals... maybe it should be the last one? before returning empty